website/meta/universe.json

```diff
@@ -1,5 +1,33 @@
 {
     "resources": [
+        {
+            "id": "whatlies",
+            "title": "whatlies",
+            "slogan": "Make interactive visualisations to figure out 'what lies' in word embeddings.",
+            "description": "This small library offers tools to make visualisation easier of both word embeddings as well as operations on them. There a convenient API to perform linear algebra as well as support for popular transformations like PCA/UMAP/etc.",
```

Comment:
A verb is missing: "There a". Perhaps it also makes sense to include in the description how it uses spaCy? Is it mainly to use the prebuilt models ?

Reply:
Ah yes. Dyslexia strikes again. Should be fixed now.